src/Appwrite/Extend/Exception.php

```diff
@@ -106,9 +106,16 @@ class Exception extends \Exception
     public const STORAGE_INVALID_CONTENT_RANGE     = 'storage_invalid_content_range';
     public const STORAGE_INVALID_RANGE             = 'storage_invalid_range';
 
+    /** VCS */
+    public const INSTALLATION_NOT_FOUND              = 'installation_not_found';
+    public const REPOSITORY_NOT_FOUND                = 'repository_not_found';
+    public const VCS_REPOSITORY_NOT_FOUND            = 'vcs_repository_not_found';
+    public const VCS_CONTRIBUTION_ALREADY_AUTHORIZED = 'vcs_contribution_already_authorized';
```

Comment:
This should probably be some kind of a conflict, we can use a better more standard name.

Reply:
Used PROVIDER_CONTRIBUTION_CONFLICT